scripts/mysqld_safe.sh

```diff
@@ -937,7 +949,12 @@ fi
 #  ulimit -n 256 > /dev/null 2>&1		# Fix for BSD and FreeBSD systems
 #fi
 
-cmd="`mysqld_ld_preload_text`$NOHUP_NICENESS"
+if [ $simulate -eq 0 ]
+then
+  cmd="`mysqld_ld_preload_text`$NOHUP_NICENESS"
+else
+  cmd=''
+fi
```

Comment:
I'd suggest to write it as something like:
cmd="`mysqld_ld_preload_text`$NOHUP_NICENESS"
[ $simulate -eq 0 ] && cmd=''

Just to reduce number of affected lines.